src/main/java/org/skriptlang/skript/bukkit/itemcomponents/blocking/elements/ExprBlockCompDisableScale.java

```diff
@@ -0,0 +1,88 @@
+package org.skriptlang.skript.bukkit.itemcomponents.blocking.elements;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.blocking.BlockingExperimentalSyntax;
+import org.skriptlang.skript.bukkit.itemcomponents.blocking.BlockingWrapper;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+@Name("Blocking Component - Disable Cooldown Scale")
+@Description("""
+	The scalar applied to the disabled cooldown time for the item when disabled by an attack.
```

Comment:
```suggestion
	A scaling multiplier for how long this item can be disabled for.
	When disabled by an attack, this item will be forced on cooldown according to the weapon's 'disable blocking' duration. That duration is multiplied by this value to get the actual cooldown duration. 
	This could be used, for example, to make variants of shields with different 'weights' that recover faster or slower from the same disabling attack.
```